one build failure:
```
16:11:21 [ant:checkstyle] [ERROR] /home/jenkins/jenkins-slave/workspace/kafka-pr-jdk11-scala2.13@2/clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java:3799:90: '{' is not preceded with whitespace. [WhitespaceAround]
```

--- a/clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java
+++ b/clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java
@@ -3796,7 +3796,7 @@ public class KafkaAdminClient extends AdminClient {
                         OffsetSpec offsetRequestSpec = topicPartitionOffsets.get(tp);
                         if (offsetRequestSpec == null) {
                             future.completeExceptionally(new KafkaException("Unexpected topic partition " + tp + " in broker response!"));
-                        } else if (MetadataOperationContext.shouldRefreshMetadata(error)){
+                        } else if (MetadataOperationContext.shouldRefreshMetadata(error)) {
                             retryTopicPartitionOffsets.put(tp, offsetRequestSpec);
                         } else if (error == Errors.NONE) {
                             future.complete(new ListOffsetsResultInfo(partitionData.offset, partitionData.timestamp, partitionData.leaderEpoch));

